### PR TITLE
fix: CLIN-2000 added missing css for hidden tabs

### PR DESCRIPTION
**Pull request by ethienneroy**
# FIX : TITLE

- closes #clin-2000

## Description
- Switching tabs wasn't hiding the previous one, they were showing side by side

[JIRA LINK]https://ferlab-crsj.atlassian.net/browse/CLIN-2000

Acceptance Criterias
- switching tabs hides the previous one
## Validation

- [ ] Code Approved
- [x] QA Done
- [ ] Design/UI Approved from design

### Before
image

### After
image

## QA

Steps to validate
Url (storybook, ...)
...

## Mention

@kstonge @luclemo

### CODE REVIEW

**Comment by ethienneroy** [comment]
@kstonge les tests qui failent sont fixer dans une autre PR qui n'a pas été merger encore

**Review by atoulous — APPROVED** [review]

**Review by francisl — CHANGES_REQUESTED** [review]

File: src/components/Layout/FixedSizeTable/index.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
-import { useRef } from 'react';
+import { useEffect, useRef, useState } from 'react';
 
-import { ITableDimension, useStaticTableHeight } from 'components/Table/StaticTableHeightXY';
+import { ITableDimension } from 'components/Table/StaticTableHeightXY';
+
+import useElementSize from '../../../hooks/useElementSize';
+import useWindowSize from '../../../hooks/useWindowSize';
```
[the remark this answers is not in the document]
Comment (ethienneroy) [reply src/components/Layout/FixedSizeTable/index.tsx:6]: absolument !

File: src/components/Table/StaticTableHeightXY.tsx [hunk 2]
```diff
@@ -2,8 +2,8 @@ import { useEffect, useState } from 'react';
 import debounce from 'lodash/debounce';
 
 export interface ITableDimension {
-  x: number;
-  y: number;
+  x: number | string;
+  y: number | string;
```
[the remark this answers is not in the document]
Comment (ethienneroy) [reply src/components/Table/StaticTableHeightXY.tsx:6]: c'etait pour suivre le scroll props de antd/table, https://ant.design/components/table#scroll

**Review by francisl — APPROVED** [review]
🍻

**Pull request merged**